identifiers/country-us/us-education-boards.csv

```diff
@@ -0,0 +1,59 @@
+id,name
+ocd-division/country:us/state:ks/board_of_education:1,Kansas State Board of Education District 1
```

Comment:
It looks like in other files in US we use "school_board_district" instead of "board_of_education" for the type. We should change to "school_board_district" for consistency

Reply:
All set 👍